modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,156 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+    load_boston,
+)
+
+
+@pytest.mark.parametrize(
+    "dataset",
+    [
+        load_iris(),
+        load_diabetes(),
+        load_digits(),
+        load_wine(),
+        load_breast_cancer(),
+        load_boston(),
+    ],
+    ids=[
+        "load_iris",
+        "load_diabetes",
+        "load_digits",
+        "load_wine",
+        "load_breast_cancer",
+        "load_boston",
+    ],
+)
+def test_dmatrix_sklearn_datasets(dataset):
```

Comment:
What does this test check? Sklearn datasets mainly should use for check correctness of train/predict.

Reply:
This test was deleted